dev/ci/ci-basic-overlay.sh

```diff
@@ -226,3 +226,18 @@
 : "${quickchick_CI_REF:=master}"
 : "${quickchick_CI_GITURL:=https://github.com/QuickChick/QuickChick}"
 : "${quickchick_CI_ARCHIVEURL:=${quickchick_CI_GITURL}/archive}"
+
+########################################################################
+# menhirlib
+########################################################################
+: "${menhirlib_CI_REF:=master}"
+: "${menhirlib_CI_GITURL:=https://gitlab.inria.fr/fpottier/coq-menhirlib}"
+: "${menhirlib_CI_ARCHIVEURL:=${menhirlib_CI_GITURL}/-/archive}"
+
+########################################################################
+# aac-tactics
+########################################################################
+# Note: this is the latest commit of the v8.8 branch as of August 31st 2018
+: "${aactactis_CI_REF:=master}"
```

Comment:
Is the comment a survivor from a 8.8-only PR?

Reply:
@SkySkimmer : yes, thanks!

Please note that this PR is not yet ready for review. I still need some experimenting what works in CI and what not (which is unfortunately not the same as what works locally - something I am researching). There are still a few things missing, among others a final cleanup, but also the dependency checking.

Maybe we should have a "work in progress" label ... I really don't know how to handle these CI centric PRs in a good way.

Reply:
>Maybe we should have a "work in progress" label ... I really don't know how to handle these CI centric PRs in a good way.

See `needs: progress` which I just put on this PR.